youtube_dl/extractor/ccma.py

```diff
@@ -52,12 +52,22 @@ def _real_extract(self, url):
             if md:
                 media_data = md
                 media_url = media_data.get('media', {}).get('url')
-                if media_url:
+                if media_url and isinstance(media_url, list):
+                    q = 0
+                    for _url in media_url:
+                        formats.append({
+                            'format_id': profile,
+                            'url': _url['file'],
+                            'quality': q,
```

Comment:
No, parse height from label.

Reply:
I'm sorry, but i don't understand what do you mean by height and label, could you elaborate?

Reply:
`_url['label']`.